### PR TITLE
Bugfix/6727 import http client module

### PR DESCRIPTION
**Pull request by macjohnny**
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).
- [x] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [x] Filed the PR against the correct branch: `3.0.0` branch for changes related to OpenAPI spec 3.0. Default: `master`.
- [x] Copied the [technical committee](https://github.com/swagger-api/swagger-codegen/#swagger-codegen-technical-committee) to review the pull request if your PR is targeting a particular programming langauge.

### Description of the PR

fixes https://github.com/swagger-api/swagger-codegen/issues/6727 and replaces PR https://github.com/swagger-api/swagger-codegen/pull/6729

### CODE REVIEW

**Comment by macjohnny** [comment]
cc @roni-frantchi @wing328 @sebastianhaas @bedag-moo @krizic @taxpon @kenisteward@watiko @pgrm @topce

**Comment by pgrm** [comment]
@wing328 you can mark this one also with bug and typescript

**Comment by macjohnny** [comment]
@wing328 @sebastianhaas the checks passed, how about merging?

File: samples/client/petstore/typescript-angular-v2/default/model/ApiResponse.ts [hunk 1]
```diff
@@ -23,3 +23,5 @@ export interface ApiResponse {
     message?: string;
 
 }
```
Comment (sebastianhaas) [inline samples/client/petstore/typescript-angular-v2/default/model/ApiResponse.ts]: Still wondering about those newlines

Comment (wing328) [reply samples/client/petstore/typescript-angular-v2/default/model/ApiResponse.ts]: I can try to clean it up later with a separate PR.

Comment (kenisteward) [reply samples/client/petstore/typescript-angular-v2/default/model/ApiResponse.ts]: Opened #6788 to address this :)

**Comment by macjohnny** [comment]
@wing328 can you restart the circleci test?

**Comment by macjohnny** [comment]
@wing328 the checks passed, could you please merge this?

**Pull request merged**